### DISCUSSION TITLE
Yolov8 label with 6 values rather than 5 !

### DISCUSSION THREAD

**Discussion by ghost** [ultralytics/ultralytics#1169] [General]
i have recently started using Yolov8 and have developed a working model which when passed through a video and have save_text=true gives 6 values instead 5 like this.
0 0.684115 0.800463 0.153646 0.0935185 0.575787

and this annotation is not picked up by labelimg but only by roboflow, anyone know what is the cause behind this ? or is there any other free annotation software which can read 6 readings

**Comment by glenn-jocher** [comment]
This is user error, you've passed save_conf=True, which adds confidences to your labels. If you don't want a 6th column then don't use it.
